src/muiThemeable.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Also a separate discussion - I notice a lot of projects are starting to move away from `.jsx` files to `.js` files. I'm not picky but when we do the directory reorg, that may be a good time to switch if we feel strongly the other way. Thoughts?